lfs/util.go

```diff
@@ -275,3 +275,36 @@ func ConvertCwdFilesRelativeToRepo(cwdchan <-chan string) (<-chan string, error)
 func IsWindows() bool {
 	return GetPlatform() == PlatformWindows
 }
+
+// Determine if a file/dir exists, returns IsDir() results too
+func FileOrDirExists(path string) (exists bool, isDir bool) {
```

Comment:
I'd like to see comments written to [Go convention](http://blog.golang.org/godoc-documenting-go-code). Specifically, start the comment with the name of the function, and end it with a period.

I know we're not that strict about comments in this project. Also, there are a lot of comments that could be added or reworded besides these. But if we're going to add more, I'd like them to follow conventions.